backend/app/api/main.py

```
@@ -7,10 +7,12 @@
     config,
     doc_transformation_job,
     documents,
+    llm_sts,
     login,
     languages,
     llm,
     llm_chain,
+    llm_sts,
```

Ruff (0.15.4)

[error] Redefinition of unused `llm_sts` from line 10: `llm_sts` redefined here

Remove definition: `llm_sts`

(F811)